script/dryrun

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+import glob
+import json
+import os
+from urllib.request import urlopen, Request
+
+DRY_RUN_URL = 'https://us-central1-moz-fx-data-derived-datasets.cloudfunctions.net/bigquery-etl-dryrun'
+exitcode = 0
+for sql in glob.glob('sql/*.sql'):
+    pad = ' ' * (60 - len(sql))
+    try:
+        print(sql, end=pad)
+        r = urlopen(
+            Request(
+                DRY_RUN_URL,
+                headers = {
+                    'Content-Type': 'application/json'
+                },
+                data=json.dumps({
+                    'dataset': 'telemetry',
```

Comment:
Currently not every query runs on `telemetry`. Some run in `telemetry_live`. It depends on https://github.com/mozilla/bigquery-etl/issues/189 for us to know which dataset a query runs in.

Reply:
when #265 is merged this should produce the correct dataset:
```suggestion
                    'dataset': os.path.basename(os.path.dirname(sql)),
```